libdevcrypto/CryptoPP.cpp

```diff
@@ -19,8 +19,9 @@
  * @date 2014
  */
 
-#include "CryptoPP.h"
 #include <libdevcore/Guards.h>
+#include "ECDHE.h"
+#include "CryptoPP.h"
```

Comment:
Not fixing this until Gav confirms the change in coding standards. This is the same header format for every header created < 2015.